src/lando/ui/legacy/forms.py

```diff
@@ -19,5 +19,10 @@ class UpliftRequestForm(forms.Form):
 class UserSettingsForm(forms.Form):
     """Form used to provide the Phabricator API Token."""
 
-    phab_api_token = forms.RegexField(required=False, regex="^api-[a-z0-9]{28}$")
-    reset_phab_api_token = forms.BooleanField(required=False)
+    phabricator_api_key = forms.RegexField(
+        required=False,
+        regex="^api-[a-z0-9]{28}$",
+        label="Phabricator API Key",
+    )
+    phabricator_api_key.widget.attrs.update({"class": "input"})
```

Comment:
This is because the form element is actually generated by Django. This is the conventional way to modify form elements in Django if using the builtin forms/widgets. I think in the long run we will likely need to move away from this, but this works well at this time since we don't have much customization.

Reply:
In the Flask world we can do things like [this](https://github.com/mozilla-conduit/lando-ui/blob/main/landoui/templates/treestatus/log.html#L43) to specify attributes in the HTML. If we can't do that in Django that's fine, just a little surprising. :sweat_smile: